src/ConfigProvider.php

```diff
@@ -99,6 +101,7 @@ public function getDependencies(): array
             'delegators' => [
                 Application::class                                         => [Slack\ApplicationDelegatorFactory::class],
                 AttachableListenerProvider::class                          => [
+                    Factory\SwooleEventListenerDelegator::class,
```

Comment:
This ensures the mezzio-swoole event listeners are registered with the PSR-14 implementation.